doc/rados/operations/pools.rst

```diff
@@ -94,19 +94,12 @@ To get even more information, you can execute this command with the ``--format``
 Creating a Pool
 ===============
 
-Before creating a pool, consult `Pool, PG and CRUSH Config Reference`_.  Your
-Ceph configuration file contains a setting (namely, ``pg_num``) that determines
-the number of PGs.  However, this setting's default value is NOT appropriate
-for most systems.  In most cases, you should override this default value when
-creating your pool.  For details on PG numbers, see `setting the number of
-placement groups`_
-
-For example:
-
-.. prompt:: bash $
-
-    osd_pool_default_pg_num = 128
-    osd_pool_default_pgp_num = 128
+Before creating a pool, consult `Pool, PG and CRUSH Config Reference`_. The
+Ceph central configuration database in the monitor cluster contains a setting
```

Comment:
s/in the monitor cluster//

Reply:
In this case, I think I'm going to keep "in the monitor cluster" in, at least for the time being. When all the pool-related documentation has been reviewed in 2024 and updated, I might well remove it. But for now, I think of it as scaffolding or rails for neophytes. I am open to being argued out of this position (and I am open to the most robust possible discussion of the issue of documenting pools).